CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 #### Bug fixes:
 * $PATH become empty after __rvm_unload executed [\#3847](https://github.com/rvm/rvm/pull/3847)
 * RVM incorrectly tries to install llvm 3.5 when trying to install Rubinius 3  [\#3848](https://github.com/rvm/rvm/pull/3848), 
+* RVM hardcodes number of compile threads [\#3856](https://github.com/rvm/rvm/pull/3856)
```

Comment:
it's more: enable automated detection of compilation cores for compiling movable(binary) rubies